### DISCUSSION TITLE
github desktop app

### DISCUSSION THREAD

**Discussion by AbdullahWaqarSatti** [community/community#68297] [Packages]
### Select Topic Area

Question

### Body

how can i install github desktop app on ubuntu

**Comment by Nazirkhanwazir** [comment]
link : https://desktop.github.com/
here you can find commands to run interminal:
https://github.com/kontr0x/github-desktop-install

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
